server/apps/main/templates/main/registration.html

```diff
@@ -22,11 +24,78 @@ <h3 class="section-heading">Welcome to AutSPACEs Registration</h3>
       <p class="registration-text">OpenHumans is a non-profit organisation designed to help people access and understand their personal data,
         and to help them do and share things that use that data.
         You can find out more about Open Humans on <a href="https://www.openhumans.org/about/"
-                                                      style="color: #ffff" class="stretched-link">their website</a>.
-      <p class="registration-text">We will never sell your data, and we will never share it without your consent.
-</p>
-       <a href="https://www.openhumans.org/account/login/"
-       target="_blank"><button class="btn btn-lg btn-light float-right">Register in Open Humans</button></a>
+                                                      style="color: #ffff">their website</a>.
+      <p class="registration-text">We will never sell your data, and we will never share it without your consent.</p>
+    </div>
+  </div>
+  <div class="row mt-3" style="color: #000;">
+    <div class="col-lg-3">
+      <div id="step1" class="card mr-3 mb-4">
+        <div class="card-header">
+          Step 1
+        </div>
+        <div class="card-body">
+          Select the <b>Register in Open Humans</b> button below!
+        </div>
+      </div>
+    </div>
+    <div class="col-lg-3">
+      <div id="step2" class="card mr-3 mb-4">
+        <div class="card-header">
+          Step 2
+        </div>
+        <div class="card-body">
+          Log in to your Open Humans account or create a new one
+        </div>
+      </div>
+    </div>
+    <div class="col-lg-3">
+      <div id="step3" class="card mr-3 mb-4">
+        <div class="card-header">
+          Step 3
+        </div>
+        <div class="card-body">
+          On the AutSPACEs project page, select <b>Connect project</b> and complete the authorization process
+        </div>
+      </div>
+    </div>
+    <div class="col-lg-3">
+      <div id="step4" class="card mr-3 mb-4">
+        <div class="card-header">
+          Step 4
+        </div>
+        <div class="card-body">
+          Return to the AutSPACEs page and fill in your profile
+        </div>
+      </div>
+    </div>
+    <div class="col-lg-3">
+      <div id="step5" class="card mr-3 mb-4">
+        <div class="card-header">
+          Step 5
+        </div>
+        <div class="card-body">
+          You're now ready to share your own story by selecting <b>Add Story</b> from the main menu
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="row mt-3" id="first">
+    <div class="col-lg-12">
+      <p><a href="https://www.openhumans.org/activity/autspaces/" target="_blank">
```

Comment:
Actually, this can also still be the login link, as the resulting page allows registering with OH and then takes you back to AutSPACEs at the end! It should probably jsut open in a new window/tab so that folks can keep this page around!

Reply:
Okay, I'll change it back. Does OpenHumans use the redirect header to figure out the project then? I don't know of a good way to test this while the project isn't officially active.

Reply:
Oh, now I get it. You're saying use the same link for both login and registration. It's conceptually a bit odd to have two buttons for this, but I get the motivation. That makes sense now.

Reply:
This is now done.